__tests__/app.test.js

```diff
@@ -224,3 +225,41 @@ describe("GET /api/users", () => {
     });
 });
 
+//A test battery thats ensures that the review requests are working correctly, given in date order (descending).
+describe("GET: /api/reviews", () => {
+    test("Status 200: responds with an array of objects containing the appropriate properties", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body}) => {
+            const {reviews} = body;
+            expect(reviews).toBeInstanceOf(Array);
+            expect(reviews).toHaveLength(13);
+            reviews.forEach((review) => {
+                expect(review).toEqual(expect.objectContaining(
+                    {
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        category: expect.any(String),
+                        designer: expect.any(String),
+                        owner: expect.any(String),
+                        review_body: expect.any(String),
```

Comment:
Double check the ticket for this one. We don't want to see the body as this will be a list of the reviews. If you think about it as a website, this would be like a list of blog posts and we need to click through to see the actual blog post itself :)

Reply:
Ahhh right, apologies I didn't see the removal of the review_body. On it now.